builtin/providers/google/resource_compute_subnetwork.go

```diff
@@ -91,12 +91,24 @@ func resourceComputeSubnetworkCreate(d *schema.ResourceData, meta interface{}) e
 		return err
 	}
 
+	// Network can be provided as just the name, or the full resource URL
+	network := d.Get("network").(string)
```

Comment:
Looks like this code shows up twice - could you extract this into a shared file?